src/components/SubTypeAccordion/SubTypeAccordion.js

```diff
@@ -19,7 +22,7 @@ import CollectionIcon from '../Icons/CollectionIcon';
 const SubTypeAccordion = ({
   subTypeCount,
   resourceIds,
-  selectedCollectionId,
+  activeCollectionId,
```

Comment:
should we be consistent for the prop name? `activeCollectionId` vs `collectionId` in the other components? I think `collectionId` should be sufficient.

Reply:
I did change `selectedCollectionId` > `collectionId` in some cases where `collectionId` was used, downstream.